ddtrace/appsec/processor.py

```diff
@@ -68,6 +103,13 @@ def __attrs_post_init__(self):
                 log.warning("[DDAS-0005-00] WAF initialization failed")
                 raise
 
+    def setup(self, gateway):
```

Comment:
any reason to not do this in the initializer? It seems based off of the usage that we initialize and then call `setup` right away.

Reply:
Ack, will do